.github/workflows/build.yaml

```diff
@@ -1,6 +1,10 @@
 name: build
 
-on: [pull_request]
+on:
```

Comment:
does this just check that master compiles? why? CI wouldnt let u merge if ur gonna break master no?

Reply:
yea it runs the pipeline again on master every day at midnight utc. This is something that most other open source repos do. This way, we can detect other failure earlier. In our case, it could tell us when a github action starts failing (maybe there's like an external resource dependency or smth). lets notice faster if ci happens to break (which should be rarely but it does happen and should be remedied immediately)